packages/node_modules/@webex/common/src/constants.js

```diff
@@ -67,3 +67,5 @@ export const hydraTypes = {
   ROOM: 'ROOM',
   TEAM: 'TEAM'
 };
+
+export const INTERNAL_US_CLUSTER_NAME = 'urn:TEAM:us-east';
```

Comment:
This will fail if they add a cluster called `urn:TEAM:us-east-something-else`. I don't know how likely this will be.

Reply:
@mwegman  how much precision do you recommend?   When we chatted on Thursday I got the impression that sometimes there were "two colons" worth of precision and sometimes three.  I went down this path to protect against NOT properly using the shorthand "us" for the ACHM cluster.    Is there a longer string that you feel will be guaranteed to be returned by the webex.internal.services.getClusterId call?

Reply:
You could compare to `urn:TEAM:us-east-1_int13` and `urn:TEAM:us-east-2_a` to cover both production and integration

Reply:
@pritomgogoi is there any chance you could point me to the hydra logic that figures out when to use the hard coded "us" in the cluster?   @mwegman's suggestion protects against future new clusters called `urn:TEAM:us-east-something-else`, but his suggestion also seems exposed to any changes to the internal names of the existing clusters.  (I'm not sure which is more likely!)

Reply:
If you change the names of existing clusters, you are going to have a lot of problems. Any existing Hydra ID will break. Anyone using the current version of the SDK will need to upgrade before we change the default cluster name.

Reply:
OK @mwegman, I have added your suggestion.